package.json

```diff
@@ -34,7 +34,7 @@
     "@bcrs-shared-components/help-business-number": "1.1.4",
     "@bcrs-shared-components/interfaces": "1.1.15",
     "@bcrs-shared-components/jurisdiction": "1.1.4",
-    "@bcrs-shared-components/limited-restoration-panel": "1.0.5",
+    "@bcrs-shared-components/limited-restoration-panel": "2.1.0",
```

Comment:
Weren't we reserving major patches for the Vue 3 versions of the components Sev?

Reply:
Yes. For this component it's 3.x.x.

What you see here is an import of a Vue2 refactor of this component (done for Edit UI) that I pulled in, plus some small changes you saw in that earlier PR.